### PR TITLE
Migrate buster

### PR DESCRIPTION
**Pull request by tricovictor**
- Group and user were created to perform the tasks.
- geerlingguy.apache is used for apache installation

### CODE REVIEW

File: tasks/copy_scripts.yml [hunk 1]
```diff
@@ -2,38 +2,38 @@
...
-    - "{{ mirror_scripts_path }}/debian/debian-cd/bin"
-    - "{{ mirror_scripts_path }}/ubuntu"
+    - debian
+    - debian/debian
+    - debian/debian/etc
+    - debian/debian/bin
+    - debian/debian-cd
+    - debian/debian-cd/etc
+    - debian/debian-cd/bin
+    - ubuntu
```
Comment (andrespias) [inline tasks/copy_scripts.yml:27]: These directories should be read from variables.

File: tasks/copy_scripts.yml [hunk 2]
```diff
@@ -2,38 +2,38 @@
...
 
 - name: Template | repository Debian
   template:
     src: ftpsync.conf.j2
     dest: "{{ mirror_scripts_path }}/debian/debian/etc/ftpsync.conf"
-    owner: root
+    owner: "{{ mirror_user.user }}"
```
Comment (andrespias) [inline tasks/copy_scripts.yml:39]: Good!

File: tasks/copy_scripts.yml [hunk 3]
```diff
@@ -1,66 +1,85 @@
...
-    - "{{ mirror_scripts_path }}/debian/debian/etc"
-    - "{{ mirror_scripts_path }}/debian/debian/bin"
-    - "{{ mirror_scripts_path }}/debian/debian-cd"
-    - "{{ mirror_scripts_path }}/debian/debian-cd/etc"
-    - "{{ mirror_scripts_path }}/debian/debian-cd/bin"
-    - "{{ mirror_scripts_path }}/ubuntu"
+    - debian
+    - ubuntu
 
 - name: Template | repository Debian
```
Comment (andrespias) [inline tasks/copy_scripts.yml:35]: We could group some copy and template tasks by parameterizing a bit more, don't you think?

File: tasks/copy_scripts.yml [hunk 4]
```diff
@@ -1,66 +1,85 @@
...
   template:
     src: ftpsync.conf.j2
     dest: "{{ mirror_scripts_path }}/debian/debian/etc/ftpsync.conf"
-    owner: root
+    owner: "{{ mirror_user.user }}"
     mode: 0766
 
 - name: Template | repository Debian-cd
```
Comment (andrespias) [inline tasks/copy_scripts.yml:42]: We could group some copy and template tasks by parameterizing a bit more, don't you think?

File: tasks/copy_scripts.yml [hunk 5]
```diff
@@ -1,66 +1,85 @@
...
     - "{{ mirror_scripts_path }}/debian/debian-cd/bin/ftpsync"
 
-- name: Copy mirror.sh
+- name: Copy notify telegram
+  template:
+    src: 'notify_telegram.sh.j2'
+    dest: "{{ mirror_scripts_path }}/notify_telegram.sh"
+    owner: "{{ mirror_user.user }}"
+    mode: 0766
+  when: mirror_telegram_token
```
Comment (andrespias) [inline tasks/copy_scripts.yml]: I get un error in this line. Please change to something like this: 
`when: mirror_telegram_token != ""`

File: tasks/copy_scripts.yml [hunk 6]
```diff
@@ -43,20 +43,20 @@
...
-    owner: root
+    owner: "{{ mirror_user.user }}"
     mode: 0766
 
 - name: Page init
   template:
     src: index.html.j2
-    dest: /var/www/html/index.html
-    owner: root
+    dest: /var/www/html/mirror/index.html
```
Comment (andrespias) [inline tasks/copy_scripts.yml]: This path need to be parameterized, don't you think?

File: tasks/copy_scripts.yml (hunk 1 above)
Comment (tricovictor) [reply tasks/copy_scripts.yml:27]: If it seems fine to me

File: tasks/copy_scripts.yml (hunk 6 above)
Comment (tricovictor) [reply tasks/copy_scripts.yml]: Parameterized var mirror_page_init

File: tasks/copy_scripts.yml (hunk 5 above)
Comment (andrespias) [reply tasks/copy_scripts.yml]: This is not solved yet

**Pull request merged**